Base/QTGUI/qSlicerStyle.cxx

```diff
@@ -105,6 +107,12 @@ QStyle::SubControl qSlicerStyle::hitTestComplexControl(ComplexControl cc, const
 int qSlicerStyle::pixelMetric(PixelMetric metric, const QStyleOption * option,
                               const QWidget * widget)const
 {
+  double pixelRatio = 1.0;
+  if (qSlicerApplication::application()->mainWindow())
+    {
+    pixelRatio = qSlicerApplication::application()->mainWindow()->devicePixelRatioF();
```

Comment:
Is this the only way to get the pixel ratio ? If possible, I would like to avoid the dependency on the application header.